docs/understand/enterprise/index.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Adding comment randomly here. 
The user journey is from OSS to Enterprise. 
Can you some minimal section about upgrades?

Reply:
@Isan-Rivkin that's a good idea but probably beyond the scope here. can we open an issue?

Reply:
I will

Reply:
https://github.com/treeverse/product/issues/310

Reply:
@itaiad200 it's in the "email" 😉 
We can do what @ozkatz suggested. 
The current version already provide tons of value .